lib/movable_ink/aws/route53.rb

```diff
@@ -13,18 +13,20 @@ def resource_record_sets(hosted_zone_id, client = nil)
       end
 
       def get_resource_record_sets_by_instance_name(zone, instance_name, client = nil)
-        resource_record_sets(zone, client).select{|rrs| rrs.set_identifier == instance_name}.first.to_h
+        resource_record_sets(zone, client).select{|rrs| rrs.set_identifier == instance_name}.map(&:to_h)
```

Comment:
Interesting, according to this bug, we originally implied that certain zone may have only one record for the instance that we're terminating. That's why it didn't hit us before. Thanks for fixing it!